src/components/navbar-dk.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+
+import WhiteLogo from '../../static/img/logos/new-white-logo.svg'
+
+export default () => (
+  <div className='container'>
+    <nav className='navbar navbar-expand-lg navbar-dk'>
```

Comment:
I assume this is `navbar-dark`? Maybe instead of using `dk`, we can just use the full name since we already have `navbar-light` in place?

Reply:
Yeah, that makes more sense. I was following Bootstrap naming conventions, but I agree with you on this. `dark` it shall be. I will also update that any where else I find it to keep it consistent throughout. Let me know if/when you see me use `dk` anywhere else too please.

Thanks!